mis_builder/models/mis_report_style.py

```diff
@@ -188,7 +189,7 @@ def render_pct(self, lang, value, dp=1, sign='-'):
     def render_str(self, lang, value):
         if value is None or value is AccountingNone:
             return u''
-        return unicode(value)
+        return str(value)
```

Comment:
It looks like this would be the only place where we'd need the future module. In which case we can find a way to do without it.

Reply:
I'd leave `unicode()` here, and at the top of the file, do

```python
if sys.version_info.major >= 3:
    unicode = str
```

Reply:
And remove the dependency on `future`